db/builder.cc

```diff
@@ -173,12 +173,28 @@ Status BuildTable(
             prev_key.assign(keys.front());
             ok = ParseInternalKey(Slice(prev_key), &prev_ikey);
             assert(ok);
+
+            // The front item has the smallest seqno in the merge queue
```

Comment:
Yes, in the iterator, “seqno within the same key are in decreasing order”. But in the merge deque, seqno are in increasing order, refer to <https://github.com/facebook/rocksdb/blob/master/db/merge_helper.cc#L153>.